styles/style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&family=Roboto+Mono&display=swap');
+
+body,
+input,
+textarea,
+button {
+  margin: 0;
+  padding: 0;
+  font-family: 'Indie Flower', monospace;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10rem;
+  gap: 1rem;
+}
+
+.books,
+fieldset,
+input,
+button {
+  border: 2px solid #000;
+}
+
+.book-info {
```

Comment:
- [ ] You did a good job on the functionalities but you need to fix the below design issues.
  - Please make sure to replace the heading `Awsome Books` with `All awesome books`.
  - Please make sure the remove book button is aligned at the end of the block even if the book title and author names are very less or big in length.
  - Please make sure to replace the heading `Add Books` with `Add a new book` and aligned it in the center.
  - Please make sure the submit button is aligned at the end instead of the center.

### Your design

![image](https://user-images.githubusercontent.com/83803414/234571282-c3c80785-ad54-48b4-b174-dab81254578f.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/234571322-bd6919cb-0ad7-4b72-9784-89d98e304681.png)